easybuild/easyconfigs/p/Python/Python-2.7.10-GNU-4.9.3-2.25-bare.eb

```diff
@@ -0,0 +1,31 @@
+name = 'Python'
+version = '2.7.10'
+versionsuffix = '-bare'
+
+homepage = 'http://python.org/'
+description = "Python is a programming language that lets you work more quickly and integrate your systems more effectively."
+
+toolchain = {'name': 'GNU', 'version': '4.9.3-2.25'}
+toolchainopts = {'pic': True, 'opt': True, 'optarch': True}
+
+source_urls = ['http://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+
+# python needs bzip2 to build the bz2 package
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('zlib', '1.2.8', '', ('GCC', '4.9.3-binutils-2.25')),
+    ('libreadline', '6.3'),
+    ('ncurses', '6.0'),
```

Comment:
this is going to be a bit weird, since we already have a `foss/2015b` and `intel/2015b` easyconfig for Python 2.7.10 where we're still using `ncurses/5.9`... :-/